src/dbus/clientprivate.cpp

```diff
@@ -416,28 +340,29 @@ void Ngf::ClientPrivate::requestEventState(Event *event, EventState wantedState)
     qCDebug(m_log) << event->clientEventId << "set state" << event->wantedState;
 
     switch (event->wantedState) {
-        case StatePlaying:
-            m_iface->asyncCall(MethodPause, event->serverEventId, QVariant(false));
-            break;
-
-        case StatePaused:
-            m_iface->asyncCall(MethodPause, event->serverEventId, QVariant(true));
-            break;
+    case StatePlaying: {
+        QDBusMessage pause = createMethodCall(MethodPause);
+        pause << event->serverEventId << QVariant(false);
 
-        case StateStopped:
-            m_iface->asyncCall(MethodStop, event->serverEventId);
-            break;
+        QDBusConnection::systemBus().asyncCall(pause);
+        break;
+    }
```

Comment:
nit: existing code etc, but this kind of switch-case block indent is confusing....

Reply:
According to convention https://wiki.qt.io/Qt_Coding_Style#Switch_statements